Source/WebCore/editing/ApplyStyleCommand.cpp

```diff
@@ -929,10 +929,8 @@ void ApplyStyleCommand::replaceWithSpanOrRemoveIfWithoutAttributes(HTMLElement&
 {
     if (hasNoAttributeOrOnlyStyleAttribute(element, StyleAttributeShouldBeEmpty))
         removeNodePreservingChildren(element);
-    else {
-        RefPtr newSpanElement = replaceElementWithSpanPreservingChildrenAndAttributes(element);
-        ASSERT_UNUSED(newSpanElement, newSpanElement && newSpanElement->isConnected());
```

Comment:
I don't think we want to remove this assertion.

Reply:
So 'RESOLVED INVALID'?